test/views/summary/correctReturn/SmallProducerDetailsSummarySpec.scala

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views.summary.correctReturn
+
+import base.SpecBase
+import models.{SelectChange, SmallProducer, UserAnswers}
+import play.api.libs.json.Json
+import uk.gov.hmrc.govukfrontend.views.Aliases.SummaryList
+
+class SmallProducerDetailsSummarySpec extends SpecBase {
+  val smallProducerList: List[SmallProducer] = List(SmallProducer("Super Cola Plc", "XCSDIL000000069", (20, 10)))
```

Comment:
Is there any value in doing a test for more than 1 producer? If there would be any value added, might be good to have one. But if not, leave as is :)

Reply:
I only did 1 producer in the controller because the controller doesn't control the detail list itself - I did include 2 in the view test